packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -111,44 +133,67 @@ export default {
         getRatingStarPercentage () {
             return `${(this.starRating / this.maxStarRating) * 100}%`;
```

Comment:
I know this is from a different PR, but would it make more sense to have this return a number, and then add the percentage sign in the style tag?